packages/@lwc/engine-server/src/__tests__/fixtures/style/expected.html

```diff
@@ -0,0 +1,7 @@
+<x-style>
+    <template shadowrootmode="open">
+    <span>
+      X-STYLE
```

Comment:
This doesn't seem to be the correct expected value. This is the `tagName`, not the style.

For `engine-server`, I think it would be fine to just throw an "unsupported method" error for the case of `getStyle`. It doesn't make much sense to me to be able to manipulate the `style` object on the server – plus, it would get very complicated to emulate the entire [CSSStyleDeclaration](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration) object on the server side.

See `engine-server/src/renderer.ts` for other examples of `unsupportedMethod()`.

Reply:
So that would mean that inside the getStyle function in `engine-server/src/renderer.ts `  I need to throw the unsupported method error.  Does this mean that I should just remove `__tests__/fixtures/style/` as we don't need getStyle in the engine-server? If not, what am I supposed to do with it?

Reply:
@le-incroyable1-dev We already have an existing test for unsupported `engine-server` methods. You can add `this.style` there:

https://github.com/salesforce/lwc/blob/fe90773a72d28038677516bc855c0ea149f56b0d/packages/%40lwc/engine-server/src/__tests__/fixtures/methods-unsupported/modules/x/methods-unsupported/methods-unsupported.js#L3-L12